ntbs-integration-tests/LabResultsPage/LabResultsPageTests.cs

```diff
@@ -16,7 +16,7 @@ public class LabResultsPageTests : TestRunnerBase
         [Fact]
         public async Task NhsUser_CanViewSpecimensAccordingToPermissions()
         {
-            using (var client = Factory.WithMockUserService<TestNhsUserService>()
+            using (var client = Factory.WithUser<NhsUserForAbingdonAndPermitted>()
```

Comment:
Not to tout my own trumpet too much here, but I do really like how this reads now...
... with the exception of the `Permitted` bit 😂 Should we just give that tb service the correct name? how bad would it affect the readability of tests where the `permitted` had a meaning?